infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogDataStore.java

```diff
@@ -0,0 +1,106 @@
+package org.corfudb.infrastructure.log;
+
+import lombok.Builder;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.infrastructure.IDataStore;
+import org.corfudb.infrastructure.IDataStore.KvRecord;
+import org.corfudb.runtime.view.Address;
+
+import java.util.Optional;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Data access layer for StreamLog
```

Comment:
This class comment is not very accurate. Please add more info about how this only stores metadata about log markers. Also, why isn't this on the LogUnit level (i.e. LogUnitMetaDataStore)?

Reply:
Added comments.

P.S. this DS manages only stream log related variables, I believe it's not quite right to use for LogUnit purposes.